source/includes/_tipser-script.md

```diff
@@ -444,7 +446,47 @@ An element with text explaining legal terms of the purchase.
 
 Tipser Elements follows <a href="https://semver.org/" target="_blank">Semantic Versioning</a>. This means that an increase in the major number in our version indicates potential <b>breaking changes</b>. Please be aware of that! For the react version, it is recommended to auto-update to the latest version with the same major number (see the caret (^) character in `package.json` file described <a href="https://stackoverflow.com/a/22345808" target="_blank">here</a>).
 
-<aside class="warning">Be aware, that tipser-elements (non-react) is always updating to the newest version published, even if the increased major version is introduced.</aside>
+<aside class="warning">Be aware, that when <code>latest.js</code> version of Tipser Script is always updating to the most recent version, even if the increased major version is introduced. To avoid that, you can fix your implementation to the specific version of Tipser Script, e.g. by using URL like <code>https://cdn.tipser.com/tipser-script/2.2.14.js</code></aside>
```

Comment:
Be aware, that when <code>latest.js</code> version =>
Be aware, that <code>latest.js</code> version